pyproject.toml

```diff
@@ -118,5 +118,15 @@ checks = ["all", #report on all checks, except the following
 exclude = ['__init__' ] # don't report on objects that match any of these regex
 
 [tool.mypy]
+disable_error_code = ['type-arg', 'import-untyped', 'import-not-found', 'no-untyped-call']
```

Comment:
Very minor nit, but could you alphabetize the list, and the variables here would probably be helpful too.